CovertInfiltration/Src/CovertInfiltration/Classes/X2StrategyElement_InfiltrationActions.uc

```diff
@@ -0,0 +1,123 @@
+// 
+
+class X2StrategyElement_InfiltrationActions extends X2StrategyElement;
+
+static function array<X2DataTemplate> CreateTemplates()
+{
+	local array<X2DataTemplate> CovertActions;
+	
+	CovertActions.AddItem(CreateGOpSuppliesTemplate());
+	CovertActions.AddItem(CreateGOpIntelTemplate());
+
+	return CovertActions;
+}
+
+static function X2DataTemplate CreateGOpSuppliesTemplate()
+{
+	local X2CovertActionTemplate Template;
+
+	`CREATE_X2TEMPLATE(class'X2CovertActionTemplate', Template, 'CovertAction_GOpSupplies');
+
+	Template.ChooseLocationFn = class'X2StrategyElement_DefaultCovertActions'.static.ChooseRandomContactedRegion;
+	Template.OverworldMeshPath = "UI_3D.Overwold_Final.GorillaOps"; // Yes, Firaxis did in fact call it Gorilla Ops
+	Template.bGoldenPath = true;
```

Comment:
Why `bGoldenPath = true;`?

Reply:
In a futile attempt to try and get the new CAs to show up in the list. I'm trying to find the CA spawning logic now so I can figure out why they are not appearing.